nemo_skills/evaluation/evaluator/ioi.py

```
@@ -439,53 +368,25 @@ async def _evaluate_entry(self, entry: dict) -> dict:
+        with open(jsonl_file, "r", encoding="utf-8") as f:
+            all_samples = [json.loads(line) for line in f]
 
-            tasks = [self._evaluate_entry(s) for s in all_samples]
-            outputs = await asyncio.gather(*tasks)
+        tasks = [self._evaluate_entry(s) for s in all_samples]
+        outputs = await asyncio.gather(*tasks)
 
-            for s, o in zip(all_samples, outputs):
-                s["test_case_results"] = o["test_case_results"]
-                s["input_case_results"] = o["input_case_results"]
+        for s, o in zip(all_samples, outputs):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)